.gitignore

```diff
@@ -6,7 +6,7 @@ yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 
-output.txt
+endpoints.md
```

Comment:
What's going on here?

Reply:
I had a random output.txt file I was using a while back and didn't want it getting in the way. endpoints.md is a local markdown file where I've listed my routes. Is that something I might want to include under docs?

Reply:
It would be useful as a precursor to API docs (OpenApi, Swagger, Postman or whatever)